src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +68,13 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      */
     public List<People> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge)
+                .filter(p -> p.getAge() <= femaleToAge
```

Comment:
```suggestion
                .filter(p -> p -> p.getAge() >= fromAge &&  (p.getAge() <= femaleToAge..))
```
you don't need two filters here